packages-ts/starknet-gauntlet-ocr2/src/commands/ocr2/deploy.ts

```diff
@@ -38,8 +38,8 @@ const makeUserInput = async (flags, args, env): Promise<UserInput> => {
       decimals: aggregator.decimals,
       description: aggregator.name,
       billingAccessController: flags.billingAccessController || env.BILLING_ACCESS_CONTROLLER || '',
-      linkToken: flags.link || env.LINK || '',
```

Comment:
so it needs to be lower case? what about `env.BILLING_ACCESS_CONTROLLER` ?

Reply:
Yup and nice catch - updated the casing for that variable as well!